content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,303 @@
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    When the RBAC policy has become effective, it should return HTTP code 403:
+
+    {{< text plain >}}
+    403
+    {{< /text >}}
+
+1.  Create a service role with name `httpbin-viewer` to give read access to
+the `httpbin` service:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRole
+    metadata:
+      name: httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      rules:
+      - services: ["httpbin.rbac-groups-test-ns.svc.cluster.local"]
+        methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+1.  Create a service role binding with name `bind-httpbin-viewer` that
+assigns the `httpbin-viewer` role to users in the `group1`.
+Wait a moment for the newly defined RBAC policy to become effective.
+Currently `request.auth.claims` is the only property that supports matching
+against a list of strings in the JWT token.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRoleBinding
+    metadata:
+      name: bind-httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      subjects:
+      - properties:
+          request.auth.claims[groups]: "group1"
+      roleRef:
+        kind: ServiceRole
+        name: "httpbin-viewer"
+    EOF
+    {{< /text >}}
+
+1.  Verify that the connection to the `httpbin`
+service succeeds after the RBAC policy has become effective.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    200
+    {{< /text >}}
+
+    When the http header includes a valid JWT with the `groups` claim
+    value [`"group1"`, `"group2"`]
+    containing `group1`, it should return HTTP code 200:
+
+    {{< text plain >}}
+    200
+    {{< /text >}}
+
+## Configure the authorization of list-typed claims
+
+Istio RBAC also supports configuring the authorization of list-typed claims.
+The example JWT also contains a JWT claim with a `scope` claim key and
+a list of strings, [`"scope1"`, `"scope2"`] as the claim value.
+You may use the `gen-jwt`
+[python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
+to generate a JWT with other list-typed claims for testing purpose.
+Instructions for using `gen-jwt.py` are in the `gen-jwt` script.
+
+1.  Create a service role binding with name `bind-httpbin-viewer` that assigns
+the `httpbin-viewer` role to a request including a JWT with the list-typed
+claim `scope` being `scope1`. Wait a moment for the newly defined RBAC policy
+to become effective.
```

Comment:
"to become effective." is not correct; you mean to say:
Wait for the newly defined RBAC policy to take effect.
Please also change it on the previous section.

Reply:
Done.